### ISSUE TITLE
[BUG] Invalid type QuantityValue

### ISSUE THREAD

**Issue by RuudvdvNB** [DivanteLtd/magento2-pimcore-bridge#16]
In the Pimcore plugin there is a mapper strategy for a QuantityValue (https://github.com/DivanteLtd/pimcore-magento2-bridge/blob/master/src/Divante/MagentoIntegrationBundle/Mapper/Strategy/MapQuantityValue.php) but when Magento fetches the product we receive the following error:

```
Error while syncing object: 2 with instance: http://domain for store view: 1. Message: An error occurred while insert/update product "2": Invalid attribute type: "quantityValue"
```

**Comment by bartoszherba** [comment]
@RuudvdvNB I am on it, there is an implementation (or lack of ii) on Pimcore side. I will cover that on magento side too.

**Comment by bartoszherba** [comment]
@RuudvdvNB This solved in #26
